mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,127 @@
+import colorlog
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import GameSerializerForImportedData, PlayerScoreSerializer
+
+FIELD_REMAPPING = {
+    "game": {"map": "Default"},
+    "player_score": {
+        "corporation": {
+            "Interpl. Cinematics": "Interplanetary Cinematics",
+            "UNMI": "United Nations Mars Initiative",
+        }
+    },
+}
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    (0, "id", int),
+    (1, "date", parser.parse),  # datetime
+    (2, "game_map", str),
+    (3, "prelude", str_to_bool),
+    (4, "colonies", str_to_bool),
+    (5, "number_of_generations", int),
+}
+
+PLAYER_SCORE_FIELD_MAPPING = {
+    (7, "player_nickname", str),
+    (8, "corporation", str),
+    (10, "terraform_rating", int),
+    (11, "milestones", int),
+    (12, "awards", int),
+    (13, "greeneries", int),
+    (14, "cities", int),
+    (15, "event_cards", int),
+    (16, "automated_cards", int),
+    (17, "active_cards", int),
+    (18, "resources", int),
+}
+
+
+# setup the logger
+handler = colorlog.StreamHandler()
+handler.setFormatter(colorlog.ColoredFormatter("%(log_color)s%(levelname)s: %(message)s"))
+
+logger = colorlog.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel("INFO")
```

Comment:
Is it okay to initialize the logger globally here?

Reply:
Yup, it is - just try doing it at the top of the file (standard)